examples/CMakeLists.txt

```diff
@@ -1,2 +1,3 @@
 add_subdirectory(forward)
 add_subdirectory(reverse)
+add_subdirectory(mixed)
```

Comment:
What is the meaning of `mixed`? What would propose as alternative names so we can decide between them?

Reply:
We can travel in both directions.